apps/gallery/test/marionette/fullscreen_image_test.js

```diff
@@ -49,19 +49,38 @@ marionette('the gallery', function() {
     assert.ok(app.thumbnailsView.displayed());
   });
 
-  test.skip('should flick through images in fullscreen mode', function() {
+  test('should flick through images in fullscreen mode', function() {
```

Comment:
nit: As we are moving away from webgl, we don't need webgl.force-enabled permission and can be removed from Line 17